ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -145,6 +145,21 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-vsphere-ovn-hybrid-custom-vxlan-port
+- as: e2e-network-migration
+  steps:
+    cluster_profile: aws
+    test:
+    - ref: ovn-sdn-migration
+    - ref: openshift-e2e-test
+    workflow: openshift-e2e-aws-ovn-sdn-migration
```

Comment:
I don't think you need a separate workflow for that, most likely `openshift-e2e-aws` would work the same

Reply:
I think we need the `-ovn-` part of this so `openshift-e2e-aws` wouldn't work. but, also `openshift-e2e-aws-ovn` doesn't work either (I think) because there isn't a need for running the e2e in this workflow.

Reply:
Oh, but I think I see what @vrutkovs saw... we don't want `e2e` in the workflow name if we are not doing e2e tests

Reply:
@vrutkovs there is  already `openshift-e2e-test` test in `openshift-e2e-aws` workflow. How can I inject my `ovn-sdn-migration` test into the workflow and ensure the `openshift-e2e-test` runs after `ovn-sdn-migration`?

Reply:
>How can I inject my ovn-sdn-migration test into the workflow

Setting a custom `test` chain in this test would replace workflow's `test` chain